### ISSUE TITLE
Can you tell me how to split the CholecSeg8k dataset? Which clips are included in the training set, test set, and verification set? 

### ISSUE THREAD

**Issue by preLeadery** [josaklil-ai/surg-active-learning#2]
Can you tell me how to split the CholecSeg8k dataset? Which clips are included in the training set, test set, and verification set?
